x-pack/plugins/security_solution/public/management/links.ts

```diff
@@ -244,18 +244,23 @@ export const getManagementFilteredLinks = async (
   plugins: StartPlugins
 ): Promise<LinkItem> => {
   const fleetAuthz = plugins.fleet?.authz;
-  const { endpointRbacEnabled, endpointRbacV1Enabled } = ExperimentalFeaturesService.get();
+  const { endpointRbacV1Enabled } = ExperimentalFeaturesService.get();
   const endpointPermissions = calculatePermissionsFromCapabilities(core.application.capabilities);
   const linksToExclude: SecurityPageName[] = [];
 
   try {
     const currentUserResponse = await plugins.security.authc.getCurrentUser();
-    const { canReadActionsLogManagement, canIsolateHost, canUnIsolateHost } = fleetAuthz
+    const {
+      canReadActionsLogManagement,
+      canIsolateHost: hasPlatinumLicense,
+      canUnIsolateHost: canIsolateHost,
```

Comment:
This variable naming does not make sense to me and its confusing. Why are you renaming them?

Also, if you need to know the license, just get it directly from the license service by importing `licenseService` directly from `https://github.com/elastic/kibana/blob/dfe8637c52d1c270d8eb783a733e6ea7c3aa1a1c/x-pack/plugins/security_solution/public/common/hooks/use_license.ts#L10`

Reply:
Actually, we only need `canIsolateHost` here. I renamed these to being able to distinguish tests where we test with `canIsolateHost` is `false` due to privilege or due to license.

Reply:
undid the renaming 8130caa06366943913c0c4bbd8c7a1c6970c404e

Reply:
undid the renaming 8130caa06366943913c0c4bbd8c7a1c6970c404e